kuma/core/decorators.py

```diff
@@ -44,6 +47,9 @@ def _cache_controlled(request, *args, **kw):
     return _shared_cache_controller
 
 
+beta_shared_cache_control = shared_cache_control(s_maxage=HOUR)
```

Comment:
My reasoning for not creating a new setting for the `beta` domain endpoints, like `CACHE_CONTROL_DEFAULT_SHARED_MAX_AGE`, is that the `beta` domain is a transition phase, and eventually `beta_shared_cache_control` would just become `shared_cache_control` and we'd use the `CACHE_CONTROL_DEFAULT_SHARED_MAX_AGE` setting (and the current `wiki` endpoints would no longer be decorated with `shared_cache_control`).

Reply:
I think in a separate future PR we should decide what we want to do with the use of `beta` in filenames and code. From the start, my thought has been that it would eventually be replaced with something else. We could use a different name, as you said something like `read_only` or `react`, or we could even drop it and rename all of the other URL filenames from `urls.py` to `urls_wiki.py` and the `urls_beta.py` filenames then become simply `urls.py`.

Reply:
Oh I get it! Once the dust settles, the wiki site will be vastly simplified as it'll just be a plain Django app without any fancy attempts to CDN and stuff.